torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -1742,8 +1740,9 @@ void ProcessGroupNCCL::watchdogHandler() {
                 reinterpret_cast<uint8_t*>(&rank) + sizeof(rank));
             globalStore_->set(std::string(EXCEPTION_DUMP), vec);
             if (!shouldDump_.load()) {
-              LOG(ERROR) << logPrefix()
-                         << "First watchdog to set the dump signal.";
+              LOG(ERROR)
+                  << logPrefix()
+                  << "Broadcasting the dump signal of flight-recorder to other processes via TCPStore.";
```

Comment:
nit: "flight-recorder dump signal" seems clearer than "dump signal of flight-recorder"